src/libpriv/rpmostree-core.c

```diff
@@ -250,6 +250,7 @@ rpmostree_treespec_new_from_keyfile (GKeyFile   *keyfile,
   }
 
   BIND_STRING("jigdo");
+  BIND_STRING("jigdo-version");
   BIND_STRING("releasever");
```

Comment:
Not new, though can we do an `#undef BIND_STRING` here?

Reply:
Fixup for this and the below :arrow_down: